libs/dh/message-archive/feature-log-search/src/lib/dh-message-archive-log-search.component.html

```diff
@@ -55,86 +46,62 @@ <h1>{{ transloco('title') }}</h1>
         />
       </watt-form-field>
 
-      <watt-form-field size="normal">
-        <watt-label class="watt-text-m">{{ transloco('role') }}</watt-label>
-
-        <select
-          #searchInputRole
-          wattInput
-          [(ngModel)]="this.searchCriteria.senderRoleType"
-          name="role"
-        >
-          <option selected [ngValue]="null">
-            {{ transloco('ChooseRole') }}
-          </option>
-          <option
-            *ngFor="let roleType of this.roleTypes | keyvalue"
-            [ngValue]="roleType['key']"
-            value="{{ roleType['value'] }}"
-          >
-            {{ roleType['key'] + ' - ' + roleType['value'] }}
-          </option>
-        </select>
-      </watt-form-field>
-
       <watt-form-field size="normal">
         <watt-label class="watt-text-m">{{
-          transloco('messageId')
+          transloco("receiverId")
         }}</watt-label>
 
         <input
-          #searchInputMessageId
-          [(ngModel)]="this.searchCriteria.messageId"
+          #searchInputReceiverId
+          [(ngModel)]="this.searchCriteria.receiverId"
           wattInput
           type="text"
-          name="messageId"
+          name="receiverId"
         />
       </watt-form-field>
 
       <watt-form-field size="normal">
         <watt-label class="watt-text-m">{{
```

Comment:
suggestion: when you use `watt-label` inside `watt-form-field` it's better to not apply additional `watt-text-*` classes. Can you remove them from here and other similar places in the file? The text in the UI will look like this: 

![image](https://user-images.githubusercontent.com/1096332/162234071-1c6c1364-4b53-47bc-a2ba-25b428c889f5.png)